json_serializable/test/supported_types/input.type_duration.dart

```diff
@@ -9,9 +9,12 @@ part 'input.type_duration.g.dart';
 @JsonSerializable()
 class SimpleClass {
   final Duration value;
+  @JsonKey(fromJson: Duration.new)
```

Comment:
these are auto-generated, so I think tests will blow-up.

LOVE that you added a test. You'll just (likely) have to add it to the kitchen sink or similar

Reply:
> these are auto-generated, so I think tests will blow-up.

I think it's fine to test with an explicit constructor without introducing custom classes for tests. Which test you're indicating will burn?

> LOVE that you added a test. You'll just (likely) have to add it to the kitchen sink or similar

Not sure what tests should be updated...? 🤔

Reply:
You will need to create a one-off test here. This will fail because you are editing a generated file!

Reply:
If I get it right, I need to update the `test/src/default_value_input.dart` with a new `@ShouldGenerate` then?